untar.c

```diff
@@ -246,13 +255,23 @@ int untar(FILE *a, char const* path)
 		}
 		else if('2' == op)
 		{
-			if(STRICT)
+			char target[101];
```

Comment:
actually, does this with with M2-Planet? I suspect it might misbehave since variable is defined in the conditional statement and M2-Planet does not know how to handle that.

Reply:
This should be resolved.